bedrock/firefox/templates/firefox/developer/whatsnew.html

```diff
@@ -25,7 +25,11 @@
       {{ high_res_img('protocol/img/logos/firefox/browser/developer/logo-word-hor-md.png', {'alt': '', 'width': '368', 'height': '68'}) }}
 
       {% if ftl_has_messages('firefox-developer-congrats-you-now-have', 'firefox-developer-welcome-to-your-new-favorite') %}
-        <h1 class="intro-title">{{ ftl('firefox-developer-congrats-you-now-have') }}</h1>
+        {% if ftl_has_messages('firefox-developer-congrats-you-now-have-latest') %}
+          <h1 class="intro-title">{{ ftl('firefox-developer-congrats-you-now-have-latest') }}</h1>
+        {% else %}
+          <h1 class="intro-title">{{ ftl('firefox-developer-congrats-you-now-have') }}</h1>
+        {% endif %}
```

Comment:
You can now simplify this conditional to:

```
<h1 class="intro-title">{{ ftl('firefox-developer-congrats-you-now-have-latest', fallback='firefox-developer-congrats-you-now-have') }}</h1>
```

Reply:
oh sure, thanks :)